packages/mux-uploader/package.json

```diff
@@ -46,7 +46,8 @@
     "publish-release": "../../scripts/publish.sh"
   },
   "dependencies": {
-    "@mux/upchunk": "^3.0.0"
+    "@mux/upchunk": "^3.0.0",
+    "sinon": "^15.0.1"
```

Comment:
Sinon must be a dev dependency otherwise it will be installed by the consumers of @mux/mux-uploader which is not needed to run this package.